shared/src/packages/projects/libs/types/project-form-data.type.ts

```diff
@@ -0,0 +1,5 @@
+type ProjectFormData = {
```

Comment:
```suggestion
type ProjectUploadImageDto = {
```
don't forget about file name